test/hotspot/jtreg/serviceability/dcmd/vm/SystemMapTestBase.java

```diff
@@ -82,14 +84,17 @@ private static class LinuxPatterns implements MapPatterns {
         static final String shouldMatchUnconditionally_linux[] = {
             // java launcher
             regexBase_committed + "/bin/java",
-            // libjvm
-            regexBase_committed + "/lib/.*/libjvm.so",
             // heap segment, should be part of all user space apps on all architectures OpenJDK supports.
             regexBase_committed + "\\[heap\\]",
             // we should see the hs-perf data file, and it should appear as shared as well as committed
             regexBase_shared_and_committed + "hsperfdata_.*"
         };
 
+        static final String shouldMatchUnconditionally_linux_libjvm[] = {
+            // libjvm
+            regexBase_committed + "/lib/.*/libjvm.so"
+        };
+
```

Comment:
Nit, misnamed, since its not unconditionally anymore.

Reply:
Done. Also renamed for other OSs.